src/test/java/de/fraunhofer/aisec/cpg/enhancements/calls/CallResolverTest.java

```diff
@@ -390,8 +390,10 @@ private void testDefaultArgumentsInDefinition() throws Exception {
     // Check defaults edge of ParamVariableDeclaration
     assertTrue(displayFunction.getDefaultParameters().get(0) instanceof Literal);
     assertTrue(displayFunction.getDefaultParameters().get(1) instanceof Literal);
-    assertEquals("*", ((Literal) displayFunction.getDefaultParameters().get(0)).getValue());
-    assertEquals("3", ((Literal) displayFunction.getDefaultParameters().get(1)).getValue());
+    assertEquals(
```

Comment:
You should use the correct type here instead of casting `toString`. In this case it is a character '*' and the number 3.